packages/drivers/routerlicious-urlResolver/.eslintrc.cjs

```diff
@@ -25,7 +25,7 @@ module.exports = {
 		],
 
 		// This library is used in the browser, so we don't want dependencies on most node libraries.
-		"import/no-nodejs-modules": ["error", { allow: ["url"] }],
+		"import/no-nodejs-modules": ["error"],
```

Comment:
@jason-ha, note to self, remove this when eslint-config-fluid is updated (as it will be the same).